app/uk/gov/hmrc/serviceconfigs/controller/NginxController.scala

```diff
@@ -78,4 +79,15 @@ class NginxController @Inject()(
         .map(routes => Json.toJson(routes))
         .map(Ok(_))
     }
+
+  @ApiOperation(
+    value = "Retrieves list of frontend services",
+    notes = """All the services in the mdtp-frontend-routes repo are defined as frontend services"""
+  )
+  def allFrontendServices(): Action[AnyContent] =
+    Action.async {
+      db.findAllFrontendServices()
```

Comment:
wondering if we should add a filter here to remove the weird results that start with $ etc?